subsys/settings/src/settings.c

```diff
@@ -234,39 +241,75 @@ int settings_commit(void)
 	return settings_commit_subtree(NULL);
 }
 
+static bool skip_commit(int ch_cprio, int cprio, int *nprio)
+{
+	if (ch_cprio > cprio) {
```

Comment:
Does it mean all priorities above minimum are lower?

Reply:
Yes, higher priority value means lower priority.